Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1707,20 +1703,22 @@ public override void UpdateCursor ()
 
 		public override void End ()
 		{
+
 			WinConsole.Cleanup ();
 			WinConsole = null;
 
+			// Disable alternative screen buffer.
+			Console.Out.Write ("\x1b[?1049l");
+
 			// Needed for Windows Terminal
 			// Clear the alternative screen buffer from the cursor to the
 			// end of the screen.
 			// Note, [3J causes Windows Terminal to wipe out the entire NON ALTERNATIVE
 			// backbuffer! So we need to use [0J instead.
 			Console.Out.Write ("\x1b[0J");
```

Comment:
This can be removed because `Console.Out.Write ("\x1b[?1047l")` will restore the terminal with the backbuffer that already existed before the app is run.